sklearn/compose/_column_transformer.py

```diff
@@ -584,6 +586,7 @@ def transform(self, X):
                                  'and for transform when using the '
                                  'remainder keyword')
 
+        # TODO: also call _check_n_features(reset=False) in 0.24
```

Comment:
you mean 0.25?

Reply:
No, `_validate_features` should raise an error from 0.24 (it's in the comments) so we can then use _check_n_features instead